module.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "forge-compendium-browser",
+  "id": "forge-compendium-browser",
   "title": "The Forge: Compendium Library",
   "description": "Browse your compendium books like they were meant to be viewed. The Forge's Compendium Library lets you navigate your D&amp;D Beyond imported books in a beautiful and easy to use interface!",
-  "version": "1.4.1",
+  "version": "1.4.0",

```

Comment:
Do we want to roll back the version? It shouldn't be a problem if we do, just want to make sure this was intentional. Adding `id` is good 👍

Reply:
Yep.  KaKaRoTo said there's no reason to go with 1.4.1, it should be 1.4.0